### DISCUSSION TITLE
Предложения по оформлению страницы репозитория

### DISCUSSION THREAD

**Discussion by maximgrynykha** [bzick/oh-my-backend#13] [Ideas]
В секцию **'About'**, возможно, имеет смысл расширить описание проекта (репозитория). Например: "Backend Roadmap (from Junior to Senior). Что нужно знать бэкенд-разработчику web-приложений.".

В секцию **'Topics'** предлагаю добавить: `roadmap`, `backend`, `middle`, `junior`, `senior`, `backend-roadmap`, `oh-my-backend`.

@bzick Буду рад услышать о ваших замечаниях по поводу предложений.

**Comment by bzick** [comment]
Внёс изменения в теги и в описание. Только в описании текст на русском вставил впереди так как это русскоязычный документ и при поиске видна только первая часть описания. Логично что бы там было на русском что бы развеять сомнения что документ на русском.
